address important win32 compile warnings

in `constructPtrRigidBodyConstraint.cpp`
```
..\..\..\..\systems\plants\constraint\constructPtrRigidBodyConstraint.cpp(10): warning C4273: 'createDrakeConstraintMexPointer' : inconsistent dll linkage [C:\cygwin\tmp\drake-precompiled-20529\drake\pod-build\systems\plants\constraint\drakeConstructRigidBodyConstraint.vcxproj]
          c:\cygwin\tmp\drake-precompiled-20529\drake\systems\plants\constraint\constructPtrRigidBodyConstraint.h(16) : see previous definition of 'createDrakeConstraintMexPointer'
..\..\..\..\systems\plants\constraint\constructPtrRigidBodyConstraint.cpp(15): warning C4273: 'rigidBodyConstraintParseTspan' : inconsistent dll linkage [C:\cygwin\tmp\drake-precompiled-20529\drake\pod-build\systems\plants\constraint\drakeConstructRigidBodyConstraint.vcxproj]
          c:\cygwin\tmp\drake-precompiled-20529\drake\systems\plants\constraint\constructPtrRigidBodyConstraint.h(17) : see previous definition of 'rigidBodyConstraintParseTspan'
..\..\..\..\systems\plants\constraint\constructPtrRigidBodyConstraint.cpp(43): warning C4273: 'rigidBodyConstraintParse3dUnitVector' : inconsistent dll linkage [C:\cygwin\tmp\drake-precompiled-20529\drake\pod-build\systems\plants\constraint\drakeConstructRigidBodyConstraint.vcxproj]
          c:\cygwin\tmp\drake-precompiled-20529\drake\systems\plants\constraint\constructPtrRigidBodyConstraint.h(18) : see previous definition of 'rigidBodyConstraintParse3dUnitVector'
..\..\..\..\systems\plants\constraint\constructPtrRigidBodyConstraint.cpp(62): warning C4273: 'rigidBodyConstraintParseQuat' : inconsistent dll linkage [C:\cygwin\tmp\drake-precompiled-20529\drake\pod-build\systems\plants\constraint\drakeConstructRigidBodyConstraint.vcxproj]
          c:\cygwin\tmp\drake-precompiled-20529\drake\systems\plants\constraint\constructPtrRigidBodyConstraint.h(19) : see previous definition of 'rigidBodyConstraintParseQuat'
..\..\..\..\systems\plants\constraint\constructPtrRigidBodyConstraint.cpp(88): warning C4273: 'rigidBodyConstraintParseGazeConethreshold' : inconsistent dll linkage [C:\cygwin\tmp\drake-precompiled-20529\drake\pod-build\systems\plants\constraint\drakeConstructRigidBodyConstraint.vcxproj]
          c:\cygwin\tmp\drake-precompiled-20529\drake\systems\plants\constraint\constructPtrRigidBodyConstraint.h(20) : see previous definition of 'rigidBodyConstraintParseGazeConethreshold'
..\..\..\..\systems\plants\constraint\constructPtrRigidBodyConstraint.cpp(109): warning C4273: 'rigidBodyConstraintParseGazeThreshold' : inconsistent dll linkage [C:\cygwin\tmp\drake-precompiled-20529\drake\pod-build\systems\plants\constraint\drakeConstructRigidBodyConstraint.vcxproj]
          c:\cygwin\tmp\drake-precompiled-20529\drake\systems\plants\constraint\constructPtrRigidBodyConstraint.h(21) : see previous definition of 'rigidBodyConstraintParseGazeThreshold'
```

see https://msdn.microsoft.com/en-us/library/35bhkfb6.aspx

--- a/systems/plants/constraint/constructPtrRigidBodyConstraint.h
+++ b/systems/plants/constraint/constructPtrRigidBodyConstraint.h
@@ -4,19 +4,19 @@
 #include <Eigen/Dense>
 
 #if defined(WIN32) || defined(WIN64)
-  #if defined(drakeRigidBodyConstraint_EXPORTS)
-    #define CONSTRAINT_DLLEXPORT __declspec( dllexport )
+  #if defined(drakeConstructRigidBodyConstraint_EXPORTS)
+    #define CONSTRUCT_CONSTRAINT_DLLEXPORT __declspec( dllexport )
   #else
-    #define CONSTRAINT_DLLEXPORT __declspec( dllimport )
+    #define CONSTRUCT_CONSTRAINT_DLLEXPORT __declspec( dllimport )
   #endif
 #else
-  #define CONSTRAINT_DLLEXPORT
+  #define CONSTRUCT_CONSTRAINT_DLLEXPORT
 #endif
 
-CONSTRAINT_DLLEXPORT mxArray* createDrakeConstraintMexPointer(void* ptr, const char* name);
-CONSTRAINT_DLLEXPORT void rigidBodyConstraintParseTspan(const mxArray* pm,Eigen::Vector2d &tspan);
-CONSTRAINT_DLLEXPORT void rigidBodyConstraintParse3dUnitVector(const mxArray* pm, Eigen::Vector3d &unit_vec);
-CONSTRAINT_DLLEXPORT void rigidBodyConstraintParseQuat(const mxArray* pm, Eigen::Vector4d &quat);
-CONSTRAINT_DLLEXPORT double rigidBodyConstraintParseGazeConethreshold(const mxArray* pm);
-CONSTRAINT_DLLEXPORT double rigidBodyConstraintParseGazeThreshold(const mxArray* pm);
+CONSTRUCT_CONSTRAINT_DLLEXPORT mxArray* createDrakeConstraintMexPointer(void* ptr, const char* name);
+CONSTRUCT_CONSTRAINT_DLLEXPORT void rigidBodyConstraintParseTspan(const mxArray* pm,Eigen::Vector2d &tspan);
+CONSTRUCT_CONSTRAINT_DLLEXPORT void rigidBodyConstraintParse3dUnitVector(const mxArray* pm, Eigen::Vector3d &unit_vec);
+CONSTRUCT_CONSTRAINT_DLLEXPORT void rigidBodyConstraintParseQuat(const mxArray* pm, Eigen::Vector4d &quat);
+CONSTRUCT_CONSTRAINT_DLLEXPORT double rigidBodyConstraintParseGazeConethreshold(const mxArray* pm);
+CONSTRUCT_CONSTRAINT_DLLEXPORT double rigidBodyConstraintParseGazeThreshold(const mxArray* pm);
 #endif

